source/index.html

```diff
@@ -1,12 +1,230 @@
 <!DOCTYPE html>
-<html lang="ru">
+<html class="page" lang="ru">
   <head>
     <meta charset="utf-8">
     <title>HTML Academy: Седона</title>
+    <link href="https://fonts.googleapis.com/css?family=PT+Sans:400,700&amp;subset=latin,cyrillic" rel="stylesheet">
+    <link rel="stylesheet" href="css/style.css">
   </head>
   <body>
-
-    <p>Репозиторий создан для обучения на профессиональном онлайн‑курсе «<a href="https://htmlacademy.ru/intensive/adaptive">HTML и CSS. Адаптивная вёрстка и автоматизация</a>».</p>
-
+    <div class="page__body-container">
+      <header class="page-header">
+        <a class="page-header__logo">
+          <img src="./img/logotype.svg" width="102" height="84" alt="логотип города Седона»">
```

Comment:
```suggestion
          <img src="./img/logotype.svg" width="102" height="84" alt="логотип города Седона.">
```
В альтах надо ставить точки на концах предложений, чтобы скрин ридер делал паузу

Reply:
Расставил точки даже там, где отдельные слова (футер). В случае с восклицательными знаками в конце предложений как быть? Менять на точки или нет?

Reply:
не, не надо. Любой знак препинания - будет пауза